WolvenKit.App/ViewModels/HomePage/Pages/ModsViewModel.cs

```diff
@@ -0,0 +1,279 @@
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+using Splat;
+using WolvenKit.Common.Services;
+using WolvenKit.Functionality.Commands;
+using WolvenKit.Functionality.Services;
+using WolvenKit.Helpers;
+using WolvenKit.Interaction;
+using WolvenKit.Models;
+using WolvenKit.ViewModels.Shell;
+
+namespace WolvenKit.ViewModels.HomePage
+{
+    public class ModsViewModel : PageViewModel
+    {
+        private readonly ISettingsManager _settings;
+        private readonly ILoggerService _logger;
+        private readonly IProjectManager _projectManager;
+        private readonly IPluginService _pluginService;
+        private readonly AppViewModel _mainViewModel;
+
+        private readonly JsonSerializerOptions _options = new JsonSerializerOptions
+        {
+            WriteIndented = true,
+            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull,
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+            IgnoreReadOnlyProperties = true,
+        };
+
+        public ModsViewModel()
+        {
+            _settings = Locator.Current.GetService<ISettingsManager>();
+            _logger = Locator.Current.GetService<ILoggerService>();
+            _projectManager = Locator.Current.GetService<IProjectManager>();
+            _pluginService = Locator.Current.GetService<IPluginService>();
+
+            _mainViewModel = Locator.Current.GetService<AppViewModel>();
+
+            RefreshCommand = ReactiveCommand.Create(() => Refresh());
+            DeployCommand = ReactiveCommand.Create(() => Deploy());
+            LaunchGameCommand = ReactiveCommand.Create(() => LaunchGame());
+            CheckRedModCommand = ReactiveCommand.Create(() => CheckRedMod());
+            OpenModFolderCommand = ReactiveCommand.Create(() => OpenModFolder());
+
+            RemoveCommand = ReactiveCommand.Create(() => Remove());
+
+            LoadMods();
+        }
+
+        [Reactive] public ObservableCollection<ModInfoViewModel> Mods { get; set; } = new();
+        [Reactive] public ModInfoViewModel SelectedMod { get; set; }
+        [Reactive] public IEnumerable<ModInfoViewModel> SelectedMods { get; set; }
+        [Reactive] public bool IsProcessing { get; set; }
+
+        #region commands
+
+        public ICommand DeployCommand { get; private set; }
+        private async Task Deploy() => await DeployRedmod();
+        private IEnumerable<ModInfoViewModel> GetEnabledMods() => Mods.Where(x => x.IsEnabled);
+        private async Task<bool> DeployRedmod()
+        {
+            if (!_pluginService.IsInstalled(EPlugin.redmod))
+            {
+                _logger.Error("Redmod needs to be installed to deploy mods.");
+                switch (await Interactions.ShowMessageBoxAsync("The RedMod tools are not installed. Would you like to install them?", "RedMod not found"))
```

Comment:
```suggestion
                var response = await Interactions.ShowMessageBoxAsync("The RedMod tools are not installed. Would you like to install them?", "RedMod not found")
                switch (response)
```
I find it easier to read this way.